packages/framesync/.gitignore

```diff
@@ -0,0 +1,61 @@
+# Logs
```

Comment:
this is covered by root `.gitignore`, u should be able to remove this entire file